### DISCUSSION TITLE
Mantine Dark Mode?

### DISCUSSION THREAD

**Discussion by juzjus10** [table-library/react-table-library#68] [Q&A] [answered]
Hi, how do I make the table support mantine's dark mode?

**Comment by rwieruch** [comment, marked as answer]
There is no pre-made theme for dark mode. However, here is what you can do:

- use the pre-made Mantine theme
- stitch it with a custom theme of yours which colors the rows dark (see https://react-table-library.com/?path=/docs/features-theme--stitch-themes)

**Reply by juzjus10** [reply]
Thank you!
